appium-dotnet-driver/Appium/Tizen/TizenDriver.cs

```diff
@@ -0,0 +1,218 @@
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the TizenDriver class using the specified remote address and desired capabilities
+        /// </summary>
+        /// <param name="remoteAddress">URI containing the address of the WebDriver remote server (e.g. http://127.0.0.1:4723/wd/hub).</param>
+        /// <param name="desiredCapabilities">An <see cref="DesiredCapabilities"/> object containing the desired capabilities.</param>
+        public TizenDriver(Uri remoteAddress, DesiredCapabilities desiredCapabilities)
+            : base(remoteAddress, SetPlatformToCapabilities(desiredCapabilities, Platform))
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the TizenDriver class using the specified Appium local service and desired capabilities
+        /// </summary>
+        /// <param name="service">the specified Appium local service</param>
+        /// <param name="desiredCapabilities">An <see cref="ICapabilities"/> object containing the desired capabilities of the browser.</param>
+        public TizenDriver(AppiumLocalService service, DesiredCapabilities desiredCapabilities)
+            : base(service, SetPlatformToCapabilities(desiredCapabilities, Platform))
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the TizenDriver class using the specified remote address, desired capabilities, and command timeout.
+        /// </summary>
+        /// <param name="remoteAddress">URI containing the address of the WebDriver remote server (e.g. http://127.0.0.1:4723/wd/hub).</param>
+        /// <param name="desiredCapabilities">An <see cref="DesiredCapabilities"/> object containing the desired capabilities.</param>
+        /// <param name="commandTimeout">The maximum amount of time to wait for each command.</param>
+        public TizenDriver(Uri remoteAddress, DesiredCapabilities desiredCapabilities, TimeSpan commandTimeout)
+            : base(remoteAddress, SetPlatformToCapabilities(desiredCapabilities, Platform), commandTimeout)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the TizenDriver class using the specified Appium local service, desired capabilities, and command timeout.
+        /// </summary>
+        /// <param name="service">the specified Appium local service</param>
+        /// <param name="desiredCapabilities">An <see cref="ICapabilities"/> object containing the desired capabilities.</param>
+        /// <param name="commandTimeout">The maximum amount of time to wait for each command.</param>
+        public TizenDriver(AppiumLocalService service, DesiredCapabilities desiredCapabilities, TimeSpan commandTimeout)
+            : base(service, SetPlatformToCapabilities(desiredCapabilities, Platform), commandTimeout)
+        {
+        }
+
+        public bool CompareImages(Image firstImage, Image secondImage, out Image resultImage)
```

Comment:
Is this method really needed? Appium provides more advanced image comparison stuff based on OpenCV: https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/image-comparison.md

Reply:
We need simple image compare function and don't want to send result image(first image).
I think this method support simple usage.

Reply:
I agree with @mykola-mokhnach . Ok, we'll update it. Thanks.

Reply:
Updated it. more review or merge plz